src/components/how-to-start/HowToStart.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import HowToCard from '../how-to-card/HowToCard';
+import content from '../../arrays/how-to-start';
+import './HowToStart.css';
+
+export default function HowToStart({ handleButtonClick }) {
+  return (
+    <section className="how">
+      <h2 className="how__title">How to start?</h2>
+      <div className="how__content">
+        <div className="how__step-container">
+          <p className="how__step">1</p>
+          <div className="how__step-line" />
+          <p className="how__step">2</p>
+          <div className="how__step-line" />
+          <p className="how__step">3</p>
+        </div>
+        {content.map((card, index) => (
+          <HowToCard
+            key={card.id}
+            step={index + 1}
+            title={card.title}
+            subtitle={card.subTitle}
+            image={card.image}
+            alt={card.alt}
+          />
+        ))}
+      </div>
+      <button type="button" className="how__button" onClick={handleButtonClick}>
```

Comment:
This button is missing a hover animation. I applied the following properties to the CTA-button in the Hero and Form component to make the button enlarge on hover. Could you apply these too?

.how__button {
  transition: all 0.3s ease;
}

.how__button:hover {
  transform: translate3D(0, -1px, 0) scale(1.05);
}

Reply:
I'll fix this in the button component branch!